images/ipfailover/keepalived/lib/failover-functions.sh

```diff
@@ -46,6 +46,13 @@ function setup_failover() {
     echo "ERROR: Module ip_vs is NOT available."
   fi
 
+  echo "  - Check for iptables rule for keepalived multicast (224.0.0.18) ..."
```

Comment:
Can we please make this behavior configurable by an ENV?  Default it to enabled.